lib/src/widgets/ensure_visible.dart

```diff
@@ -94,8 +75,8 @@ class EnsureVisibleState extends State<EnsureVisible> {
     return await position.ensureVisible(
       renderObject,
       alignment: alignment,
-      duration: widget.duration ?? duration,
```

Comment:
Why do you remove this?

Reply:
Because I removed the deprecated `widget.duration`

Reply:
@axel-op Nevermind, for some reason it looked like the whole line was removed instead of only the `widget.duration ??` part.